MusicTaste/MusicTaste/View Controllers/ProfileViewController.m

```diff
@@ -1,13 +1,30 @@
 #import "ProfileViewController.h"
+#import "UIImageView+AFNetworking.h"
+#import "User.h"
 
 @interface ProfileViewController () 
+//connects objects: Screenname and profileview
+@property (weak, nonatomic) IBOutlet UILabel *screen_name;
+@property (weak, nonatomic) IBOutlet UIImageView *profileView;
+
 
 @end
 
 @implementation ProfileViewController
-    
+
 - (void)viewDidLoad {
     [super viewDidLoad];
+    //Sets username
+    self.screen_name.text = [[User user] name];
+    //Gets image Url
+    NSString *URLString = [[User user] profilePicture];
+    NSString *stringWithoutNormal = [URLString stringByReplacingOccurrencesOfString:@"_normal" withString:@""];
+    NSURL *urlNew = [NSURL URLWithString:stringWithoutNormal];
+    [self.profileView setImageWithURL: urlNew];
+    //Makes image Circle
+    self.profileView.layer.cornerRadius = self.profileView.frame.size.height/2;
```

Comment:
Sets the iamge to a Circular image

Reply:
Yep, good example of something non-obvious that was good to comment as you did on line 24 👍